Simplenote/Classes/SPAboutViewController.swift

```diff
@@ -275,41 +318,62 @@ private extension SPAboutViewController {
+        "\\u004e\\u006f&#x74;&#x68;\\u0069\\u006e\\u0067&#x20;&#x69;&#x73;\\u0020\\u006d&#x6f;\\u0072&#x65;\\u0020\\u0073&#x69;\\u006d&#x70;&#x6c;\\u0065\\u0020\\u0074&#x68;\\u0061&#x6e;\\u0020\\u0067&#x72;\\u0065\\u0061&#x74;&#x6e;\\u0065&#x73;\\u0073\\u003b\\u0020\\u0069&#x6e;\\u0064\\u0065\\u0065&#x64;&#x2c;\\u0020\\u0074\\u006f&#x20;\\u0062\\u0065\\u0020&#x73;\\u0069\\u006d\\u0070&#x6c;\\u0065&#x20;&#x69;\\u0073&#x20;&#x74;\\u006f\\u0020&#x62;\\u0065\\u0020\\u0067&#x72;&#x65;\\u0061&#x74;\\u002e\n\\u002d&#x20;&#x52;\\u0061\\u006c&#x70;&#x68;\\u0020&#x57;\\u0061\\u006c&#x64;&#x6f;&#x20;&#x45;\\u006d&#x65;&#x72;&#x73;\\u006f&#x6e;",
+        "&#x53;\\u0069\\u006d&#x70;&#x6c;\\u0065&#x20;&#x63;&#x61;\\u006e\\u0020\\u0062&#x65;&#x20;\\u0068\\u0061&#x72;\\u0064\\u0065\\u0072\\u0020\\u0074&#x68;\\u0061\\u006e\\u0020\\u0063\\u006f\\u006d&#x70;\\u006c\\u0065&#x78;\\u002e\\u0020&#x20;\\u0059&#x6f;&#x75;&#x20;\\u0068&#x61;&#x76;&#x65;\\u0020&#x74;\\u006f\\u0020&#x77;\\u006f&#x72;&#x6b;&#x20;\\u0068&#x61;&#x72;\\u0064\\u0020\\u0074&#x6f;&#x20;&#x67;\\u0065&#x74;\\u0020&#x79;&#x6f;&#x75;&#x72;&#x20;&#x74;&#x68;\\u0069&#x6e;&#x6b;&#x69;&#x6e;\\u0067&#x20;&#x63;\\u006c&#x65;&#x61;&#x6e;&#x20;&#x74;&#x6f;\\u0020&#x6d;\\u0061&#x6b;\\u0065\\u0020&#x69;\\u0074\\u0020&#x73;&#x69;&#x6d;\\u0070\\u006c\\u0065&#x2e;&#x20;\\u0020&#x42;&#x75;&#x74;\\u0020&#x69;&#x74;\\u0027\\u0073&#x20;\\u0077\\u006f&#x72;\\u0074\\u0068&#x20;\\u0069\\u0074\\u0020&#x69;\\u006e&#x20;&#x74;&#x68;\\u0065&#x20;&#x65;\\u006e\\u0064\\u0020\\u0062\\u0065&#x63;\\u0061&#x75;\\u0073&#x65;&#x20;&#x6f;\\u006e\\u0063&#x65;&#x20;&#x79;\\u006f\\u0075&#x20;\\u0067\\u0065\\u0074\\u0020&#x74;&#x68;\\u0065&#x72;&#x65;&#x2c;\\u0020\\u0079&#x6f;&#x75;&#x20;\\u0063\\u0061\\u006e&#x20;\\u006d&#x6f;\\u0076&#x65;\\u0020\\u006d\\u006f&#x75;&#x6e;&#x74;&#x61;\\u0069&#x6e;\\u0073&#x2e;\n&#x2d;\\u0020\\u0053\\u0074&#x65;\\u0076&#x65;&#x20;&#x4a;\\u006f\\u0062&#x73;",
+        "\\u0044&#x6f;&#x20;&#x79;\\u006f&#x75;\\u0020\\u0077\\u0061&#x6e;\\u0074\\u0020&#x74;&#x6f;\\u0020\\u0077&#x6f;\\u0072&#x6b;\\u0020&#x77;&#x69;&#x74;\\u0068\\u0020\\u0075\\u0073\\u0020\\u0061\\u006e\\u0064&#x20;&#x6d;\\u0061\\u006b&#x65;\\u0020&#x74;\\u0068\\u0069\\u006e\\u0067&#x73;\\u0020&#x6c;\\u0069&#x6b;&#x65;\\u0020\\u0074\\u0068\\u0069\\u0073\\u003f\\u0020&#x45;&#x6d;\\u0061&#x69;\\u006c&#x20;\\u0075\\u0073\\u0020\\u0061\\u0074\\u0020&#x73;\\u0075&#x70;&#x70;\\u006f\\u0072\\u0074&#x40;&#x73;&#x69;&#x6d;&#x70;&#x6c;\\u0065\\u006e\\u006f&#x74;&#x65;&#x2e;&#x63;&#x6f;&#x6d;&#x20;\\u0061&#x6e;\\u0064\\u0020\\u006d\\u0065&#x6e;&#x74;\\u0069\\u006f&#x6e;&#x20;&#x74;&#x68;\\u0069\\u0073\\u0020&#x6d;\\u0065&#x73;&#x73;\\u0061&#x67;\\u0065&#x2e;"
```

Comment:
I LOVE this!!! Hooowever: these strings won't end up localized, I'm afraid.

Perhaps we should just use plain strings here?

Reply:
☝️ As discussed over Slack, please disregard!!